modules/configuring-the-codeflare-operator.adoc

```diff
@@ -14,13 +14,28 @@ ifdef::cloud-service[]
 * You have logged in to OpenShift with the `cluster-admin` role.
 endif::[]
 
-ifndef::upstream[]
-* You have enabled the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-the-distributed-workloads-components_distributed-workloads[Configuring the distributed workloads components].
-endif::[]
+
 ifdef::upstream[]
-* You have enabled the required distributed workloads components as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-the-distributed-workloads-components_distributed-workloads[Configuring the distributed workloads components].
+* You have installed the required distributed workloads components as described in link:{odhdocshome}/installing-open-data-hub/#installing-distributed-workload-components_installv2[Installing the distributed workloads components].
+endif::[]
+
+ifdef::self-managed[]
+
+ifndef::disconnected[]
+* You have installed the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components].
+endif::[]
+
+ifdef::disconnected[]
+* You have installed the required distributed workloads components as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/installing-and-deploying-openshift-ai_install#installing-distributed-workload-components_component-install[Installing the distributed workloads components].
 endif::[]
```

Comment:
Same as above. Because there is only one SM "Working with distributed workloads" book (there is not a separate one for disconnected), these links could be combined.

Reply:
Explained in DM that the module has moved from the DW guide to the installation guides.